### PR TITLE
Navbar

### PR DESCRIPTION
**Pull request by tebanwolox**
## Summary
- Create navbar in the page.

## Screenshot
Screen Shot 2019-05-17 at 15 57 56

## Trello Card
https://trello.com/c/IeoCdIJY/13-navbar

### CODE REVIEW

File: scss/navbar.scss [hunk 1]
```diff
@@ -0,0 +1,24 @@
+.nav {
+  align-items: center;
+  background-color: $white;
+  box-shadow: 5px 7px 5px 0px rgba(0,0,0,0.3);
```
Comment (nmartinelli23) [inline scss/navbar.scss]: Please use variables for colors

File: scss/navbar.scss [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  z-index: 1;
+    
+  .navbar-logo {
+    height: 100%;
+    max-height: $nav-logo-height;
+  }
+  .navbar-text {
+    font-size: $text-big;
+  }
+}
```
Comment (nmartinelli23) [inline scss/navbar.scss]: Add a last line.

File: scss/navbar.scss [hunk 3]
```diff
@@ -0,0 +1,24 @@
+.nav {
+  align-items: center;
+  background-color: $white;
+  box-shadow: 5px 7px 5px 0px $grey;
+  display: flex;
+  flex-wrap: wrap;
+  height: 100%;
```
Comment (henryzarza) [inline scss/navbar.scss]: Use `nav-height` variable

File: scss/navbar.scss [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+  justify-content: space-between;
+  left: 0;
+  max-height: $nav-height;
```
Comment (henryzarza) [inline scss/navbar.scss]: Delete this

File: scss/navbar.scss [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+  padding: 5px 20px;
+  position: fixed;
+  top: 0;
+  right: 0;
```
Comment (henryzarza) [inline scss/navbar.scss]: Delete this

File: scss/navbar.scss [hunk 6]
```diff
@@ -0,0 +1,24 @@
+.nav {
+  align-items: center;
+  background-color: $white;
+  box-shadow: 5px 7px 5px 0px $grey;
+  display: flex;
+  flex-wrap: wrap;
+  height: 100%;
+  justify-content: space-between;
+  left: 0;
```
Comment (henryzarza) [inline scss/navbar.scss]: You can replace this by `width: 100%`

File: scss/navbar.scss [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+  position: fixed;
+  top: 0;
+  right: 0;
+  z-index: 1;
+    
+  .navbar-logo {
+    height: 100%;
+    max-height: $nav-logo-height;
+  }
+  .navbar-text {
```
Comment (henryzarza) [inline scss/navbar.scss]: Missing break line

File: scss/navbar.scss [hunk 8]
```diff
@@ -0,0 +1,24 @@
...
+  left: 0;
+  max-height: $nav-height;
+  padding: 5px 20px;
+  position: fixed;
+  top: 0;
+  right: 0;
+  z-index: 1;
+    
+  .navbar-logo {
+    height: 100%;
```
Comment (henryzarza) [inline scss/navbar.scss]: Use `nav-logo-height` variable

File: scss/navbar.scss [hunk 9]
```diff
@@ -0,0 +1,24 @@
...
+    max-height: $nav-logo-height;
```
Comment (henryzarza) [inline scss/navbar.scss]: Delete this because is unnecessary

File: scss/size.scss [hunk 10]
```diff
@@ -1,4 +1,4 @@
-$nav-height: 70px;
+$nav-height: 90px;
```
Comment (henryzarza) [inline scss/size.scss]: Why you change this value?

File: scss/navbar.scss [hunk 11]
```diff
@@ -0,0 +1,23 @@
...
+  flex-wrap: wrap;
+  height: $nav-height;
+  justify-content: space-between;
+  padding: 5px 20px;
+  position: fixed;
+  top: 0;
+  width: 100%;
+  z-index: 1;
+
+
```
Comment (henryzarza) [inline scss/navbar.scss]: Delete this empty line

**Pull request closed without merging**